topics/single-cell/tutorials/scrna-case_cell-cycle/tutorial.md

```diff
@@ -128,18 +130,19 @@ The first step towards reducing the effects of the cell cycle on our dataset is
 >        - *"Format for the list of genes associated with G2M phase"*: `File`
 >            - {% icon param-file %} *"File with the list of genes associated with G2M phase"*: `g2mPhase` (Input dataset)
 >            
+> 
 > 2. Rename the output `CellCycle_Annotated`
 >
-> <question-title></question-title?
+> 
+<question-title></question-title>
 >
 > 1. Why don't we need a list of genes expressed in the G1 Phase?
 >
-> > <solution-title></solution-title>
+> > <div id="solution-1" class="box-title"><button type="button" aria-controls="solution-1-contents" aria-expanded="true" aria-label="Toggle solution box: "><i class="far fa-eye" aria-hidden="true"></i><span class="visually-hidden"></span> Solution<span role="button" class="fold-unfold fa fa-minus-square"></span></button></div>
```

Comment:
Huh, this shouldn't be necessary. Was there a bug here you were trying to fix?

Reply:
Some of them weren't looking right and the markdown tutorial said it should have this https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html

Reply:
Ahhh that's a bug on our side.